lib/idl/detector_aperture_geometry.pro

```diff
@@ -0,0 +1,147 @@
+PRO detector_aperture_geometry,g,wn,rdist,zdist,v,d,rc
+    ;+#detector_aperture_geometry
+    ;+ Defines charged fusion product diagnostic geometry
+    ;+ Sources: detector_aperture_geometry.dat and MAST_29908_221_g_ensemble.nml
+    ;+ NOTE: This file is hardcoded to be useful for developers. General users could modify parameters under
+    ;+       wn = 0 or 1, and correspondingly set the wn flag when calling detector_aperture_geometry.pro
+    ;+       Alternatively, the user could manually define the geometry, see run_tests.pro and test_cfpd.pro
+    ;+***
+    ;+##Arguments
+    ;+    **g**: GEQDSK file
+    ;+
+    ;+    **wn**: indicates Werner (1) or Netepneko (0) diagnostic geometry
+    ;+
+    ;+##Outputs
+    ;+    **rdist**: Radial coordinates of detector [m]
+    ;+
+    ;+    **zdist**: Vertical coordinates of detector [m]
+    ;+
+    ;+    **v**: Detector orientations [m/s, rad/s, m/s]
+    ;+
+    ;+    **d**: Collimator length [m]
+    ;+
+    ;+    **rc**: Outer collimator spacing [m]
+    ;+
+    ;+##Example Usage
+    ;+```idl
+    ;+IDL> g = 'g000001.01000'
+    ;+IDL> detector_aperture_geometry,g,0,rdist,zdist,v,d,rc
+    ;+```
+
+    if wn eq 1 then begin ; Source: MAST_29908_221_g_ensemble.nml
+        print, 'Using MAST geometry 1 (Werner)'
+        ;; Detector orientation
+        theta_port=fltarr(4)
+            theta_port(1-1)= 40.0

```

Comment:
why not just use 0,1,2,3 instead of pseudo 1-based indexing

Reply:
Changed it.